cypress/test/AlgoliaSearch.cy.js

```diff
@@ -0,0 +1,7 @@
+import { mount } from 'cypress/react'
+import AlgoliaSearch from '../../components/AlgoliaSearch'
+describe('AlgoliaSearch component', () => {
+    it('renders without errors', () => {
+        mount(<AlgoliaSearch />);
```

Comment:
You have to use SearchButton inside `AlgoliaSearch` to test the component. 

Also, derive a case where it searches for certain page inside docs, by typing it in the search box.